data-raw/process_data_set_lizette.R

```diff
@@ -0,0 +1,35 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+library(janitor)
+library(usethis)
+
+# cabinet_turnover --------------------------------------------------------------------
+cabinet_turnover <- read_csv("data-raw/cabinet-turnover/cabinet-turnover.csv") %>% 
+  clean_names() %>%
+  mutate(combined = str_detect(length, "combined")) %>% 
+  mutate(length = str_replace_all(length, " combined", "")) %>% 
+  mutate(length = as.numeric(length)) %>% 
+  mutate(president = factor(president))  
+
+cabinet_turnover <- cabinet_turnover %>% 
+  mutate(start = replace(start, cabinet_turnover$appointee == "Bob Gates", "12/18/2006" )) %>%
```

Comment:
Could you combine this mutate with the previous one, and instead of using `replace`, could you use `case_when()` from the dplyr package? Here is an [example](https://rudeboybert.github.io/SDS192/tips_and_tricks.html#case_when()) of it's use.

Reply:
When using `case_when()`, I am adding the correct dates to the observations that were missing dates but then all other start dates that were already in `cabinet-turnover` turn into NA, due to the `TRUE ~ NA` condition. What are your thoughts?

Reply:
For "all other start dates" you mention, don't use `TRUE ~ NA`, but rather `TRUE ~ start`. That should fill the non-specified cases with the original contents of `start`. Something like this:
```
mutate(
  start = case_when(
    appointee == "Bob Gates" ~ "12/18/2006" ,
    ..., 
    # Need this for everything else that aren't the above cases:
    TRUE ~ start
  ))
```